resources/server/players/sv_players.ts

```diff
@@ -89,6 +89,54 @@ export async function getPlayerInfo(identifier: string): Promise<PlayerQueryResp
   return { firstname, lastname, phone_number };
 }
 
+const clean = (input: string) => (input ? input.replace(/[^0-9a-z]/gi, '') : input);
```

Comment:
If anyone has some elite JS to simplify/consolidate/condense all these functions let me know!